api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/service/ContrastAccountServiceImpl.java

```diff
@@ -0,0 +1,236 @@
+        AccountValidationResponse validationResponse = new AccountValidationResponse();
+        validationResponse.setValidationStatus(FAILURE);
+        validationResponse.setMessage("Contrast validation Failed");
+        try {
+            String apiUrl = new URIBuilder(String.format(CONTRAST_URL_TEMPLATE, pluginData.getContrastEnvironmentName()))
+                    .setPath(ORGANIZATIONS_URL_PATH + pluginData.getContrastOrganizationId()).build().toString();
+            String authKey = getAuthKey(pluginData);
+            HttpGet request = new HttpGet(apiUrl);
+            request.setHeader(HttpHeaders.AUTHORIZATION, authKey);
+            request.setHeader(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON_VALUE);
+            request.setHeader("API-Key", pluginData.getContrastApiKey());
+            try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+                try (CloseableHttpResponse response = httpClient.execute(request)) {
+                    int statusCode = response.getStatusLine().getStatusCode();
+                    if (statusCode != 200) {
+                        String errorMessage = handleErrorResponse(response.getStatusLine().getStatusCode());
+                        validationResponse.setErrorDetails(errorMessage);
+                        return validationResponse;
+                    }
+                    JsonNode jsonResponse = objectMapper.readTree(response.getEntity().getContent());
+                    String organizationName = jsonResponse.get("name").textValue();
+                    if (StringUtils.isEmpty(organizationName)) {
+                        validationResponse.setErrorDetails("Couldn't read Organization details");
+                        validationResponse.setMessage("Couldn't read Organization details");
+                        return validationResponse;
+                    }
+                    pluginData.setContrastOrganizationName(organizationName);
+                    validationResponse.setValidationStatus(SUCCESS);
+                    validationResponse.setMessage("Connection established successfully");
+                    return validationResponse;
+                }
+            }
+        } catch (UnknownHostException e) {
+            LOGGER.error("Unknown host error: " + e.getMessage(), e);
+            validationResponse.setErrorDetails("The Environment name may be incorrect or the server may be down.");
+        } catch (Exception e) {
+            LOGGER.error("Unexpected error: " + e.getMessage(), e);
+            validationResponse.setErrorDetails("Unexpected error occurred");
+        }
+        return validationResponse;
+    }
+
+    private String handleErrorResponse(int statusCode) {
+        switch (statusCode) {
+            case 400:
+            case 404:
+                return "Check the Organization ID make sure it is correct.";
+            default:
+                return "Check your credentials and make sure they are correct.";
+        }
```

Comment:
Let's move this switch into place where it is used and get rid of method. This method has almost zero logic and having whole message setting thing in a single place will increase readability.